tests/decode_images.rs

```diff
@@ -236,11 +236,6 @@ fn test_tiled_oversize_gray_i8() {
     test_image_sum_i8("tiled-oversize-gray-i8.tif", ColorType::Gray(8), 1214996);
 }
 
-#[test]
-fn test_tiled_whiteiszero_gray_i8() {
-    test_image_sum_i8("tiled-whiteiszero-gray-i8.tif", ColorType::Gray(8), 1214996);
-}
-
 #[test]
```

Comment:
Is there a reason this test is being removed?

Reply:
I thought we were going to remove the WhiteIsZero conversions for i8 and i16, so I removed the test as well. (Sorry if I misunderstood)

Reply:
I read your last reply to the "signed conversions.." thread as "leave things as they are after https://github.com/image-rs/image-tiff/pull/128" but you probably meant "leave things as they are before https://github.com/image-rs/image-tiff/pull/128" ?

Reply:
Is that test failing now?